docs/dev/api.rst

```diff
@@ -66,6 +69,46 @@ The playlists API allows for the creation and editing of lists of recordings
    :include-empty-docstring:
    :undoc-static:
 
+Feedback API Endpoints
+^^^^^^^^^^^^^^^^^^^^^^
+These API endpoints allow to submit and retrieve feedback for a user's recordings
+
+.. autoflask:: listenbrainz.webserver:create_app_rtfd()
+   :blueprints: feedback_api_v1
+   :include-empty-docstring:
+   :undoc-static:
+
+Recording Recommendation API Endpoints
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ListenBrainz uses collaborative filtering to generate recording recommendations,
+which may be further processed to generate playlists for users. These api endpoints
+allow to fetch the raw collaborative filtered recording IDs.
+
+.. autoflask:: listenbrainz.webserver:create_app_rtfd()
+   :blueprints: recommendations_cf_recording_v1
+   :include-empty-docstring:
+   :undoc-static:
+
+Recording Recommendation Feedback API Endpoints
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ListenBrainz uses collaborative filtering to generate recording recommendations,
+which may be further processed to generate playlists for users. These api endpoints
+allow to submit and retrieve feedback for raw collaborative filtered recordings.
+
+.. autoflask:: listenbrainz.webserver:create_app_rtfd()
+   :blueprints: recommendation_feedback_api_v1
+   :include-empty-docstring:
+   :undoc-static:
+
+Missing MusicBrainz Data API Endpoints
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+These api endpoints allow to retrieve data which the user has submitted to
+ListenBrainz but is missing from MusicBrainz.
```

Comment:
I don't understand what this API does. Is it things that are _missing_ from MusicBrainz, or things that we've not successfully managed to link to MusicBrainz? What is the way that this is added, and why would a user want to access this data?

Reply:
This is generated during matching the listens with the MBID mapping. The listens could not be matched that are used to create this data. The intent to add this data was for displaying it in some form on LB website so as to nudge users to add this data to MB.

Speaking in general regardless of whether the API is useful to users or not, I think putting it on the docs adds a little to the convenience of developers. I say this because docs are rendered in a more human readable form on readthedocs than the source.

In case, these endpoints adds noise for users, we should probably consider adding another different section to docs.

Reply:
yeah, I'd consider adding this to a separate section not directly on the core submit/get documentation.